.storybook/styles.scss

```diff
@@ -30,6 +30,10 @@ body {
     position: absolute;
   }
 
+  .ui-side-panel__dialog {
```

Comment:
if you want "hack" things like side panel dialog to solve issue with code button do that here